src/client/src/components/Content/Schedule/Calendar/Day/styles.css

```diff
@@ -1,8 +1,8 @@
 
 .day {
-    display: table-cell;
+  display: table-cell;
 }
 
 .day li {
-    display: block;
```

Comment:
I can see that you fixed the indentation for the js files (you made it tab size 4 if I'm not mistaken).

The indentation for the css files will be 2 spaces or...?

Reply:
It's actually tab 2 everywhere, in the pull request it doesn't show but if you look in the code of GitHub it looks good.